### PR TITLE
[REVIEW] Bug dask cudf personalization

### PR DESCRIPTION
**Pull request by Iroy30**
fixes pagerank error when personalization is passed

### CODE REVIEW

**Review by jnke2016 — COMMENTED** [review]
It looks good to me

**Review by BradReesWork — APPROVED** [review]

**Review by rlratzel — APPROVED** [review]
Looks good, just one minor request.

File: python/cugraph/dask/link_analysis/pagerank.py [hunk 1]
```diff
@@ -139,13 +138,37 @@ def pagerank(input_graph,
...
 
     if personalization is not None:
-        null_check(personalization["vertex"])
-        null_check(personalization["values"])
         if input_graph.renumbered is True:
             personalization = input_graph.add_internal_vertex_id(
                 personalization, "vertex", "vertex"
             )
-        p_data = get_distributed_data(personalization)
+
```
[the remark this answers is not in the document]
Comment (Iroy30) [reply python/cugraph/dask/link_analysis/pagerank.py:145]: addressed

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**